shell/platform/linux/fl_platform_views_plugin.h

```diff
@@ -0,0 +1,73 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_PLATFORM_VIEWS_PLUGIN_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_PLATFORM_VIEWS_PLUGIN_H_
+
+#include "flutter/shell/platform/linux/fl_gesture_helper.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_binary_messenger.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_platform_views.h"
+
+G_BEGIN_DECLS
+
+G_DECLARE_FINAL_TYPE(FlPlatformViewsPlugin,
+                     fl_platform_views_plugin,
+                     FL,
+                     PLATFORM_VIEWS_PLUGIN,
+                     GObject);
+
+/**
+ * FlPlatformViewsPlugin:
+ *
+ * #FlPlatformViewsPlugin is a plugin that implements the shell side
+ * of SystemChannels.platform_views from the Flutter services library.
+ */
+
+/**
+ * fl_platform_views_plugin_new:
+ * @messenger: (transfer-none): an #FlBinaryMessenger.
+ * @gesture_helper: (transfer-none): an #FlGestureHelper for accepting gestures
+ *                                   by Dart side.
+ *
+ * Creates a new plugin that implements SystemChannels.platform_views from the
```

Comment:
This does more than handling the platform_views platform channels. This plugin manages the lifecycle of platform views and gesture handling as well.

Reply:
Done.